calc.js

```diff
@@ -1,3 +1,7 @@
 function sum(a,b) {
     return a+b;
+}
+
+function sub(a,b) {
```

Comment:
인자 여러개가 되더라도 동작할수있게 하는게 어떨까

```suggestion
function sub(...arg) {
```